### ISSUE TITLE
Update 0056-REWA spec and ACs

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/specs#1061]
The 0056-REWA spec needs to be updated in line with the new implementation:

- https://github.com/vegaprotocol/vega/issues/5273

> Now each market has it’s own reward account, they no longer compete and the reward is never split between any markets

and the following discussion:

https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1650635067593759?thread_ts=1650561120.374139&cid=CAHA5EX0F

If any addition ACs are required, or existing needing amending, please also do so under this issue.

**Comment by JonRay15** [comment]
In addition to this, I discussed with David a couple of errors I think I found in the existing spec, some of which might end up getting changed anyway due to the changes in implementation, but in the event they dont, details are below.

**Distributing maker fees received rewards**
The balance transferred to the rewards account switches from 90 VEGA to 120 VEGA in this AC.
But in the calculation you still calculate based on 90.
It should be 120 x each party's metric and therefore 59.89 for party_1 and 60.11 for party_0

![image](https://user-images.githubusercontent.com/89918817/165094704-62ff8219-6e1c-4583-a6f6-dbc763be6507.png)


**Distributing LP fees received rewards**
Again the reward pool has 120 instead of 90
I think this should be going to the USDT | sum_of_LP_fees_received | VEGA?
I think it should be referring to the metric sum_of_LP_fees_received?
(I'm assuming both of the above given the title and the fact its multiplied by 0.0003 which is the liquidity_fee)
Then I think what is going on at the end is you calculate 16.77, and then are basically saying because there is no other LP this is 100% of the LP fees received therefore party_0 gets the whole pool, but a) I think it would be better to state this definitively for avoidance of doubt, and b) it then says 90 which isnt 100% due to the 90>120 switch in the scenario.

![image](https://user-images.githubusercontent.com/89918817/165094793-be9f2dd5-e006-4c82-a6a4-b40a503c1165.png)
